libsql-sqlite3/ext/crr/Makefile

```diff
@@ -1,3 +1,5 @@
+LDLIBS += -lm
```

Comment:
I'm not sure why we need this...
@penberg - do you remember why you changed this Makefile?

Reply:
Once you compile crr with vector, it needs sqrt for cosine distance function